support-frontend/assets/pages/new-contributions-landing/components/ContributionTicker/ContributionTicker.jsx

```diff
@@ -13,6 +13,7 @@ type StateTypes = {|
 type PropTypes = {|
   // URL to fetch ticker JSON from
   tickerJsonUrl: string,
+  onGoalReached: () => void,
```

Comment:
rather than wiring this component up to the redux store, it gets a callback to use if the goal has been reached